doc/usage/configuration.rst

```diff
@@ -36,8 +34,8 @@ Important points to note:
   ``"sphinx.builders.Builder"`` means the ``Builder`` class in the
   ``sphinx.builders`` module.
 
-* Remember that document names use ``/`` as the path separator and don't contain
-  the file name extension.
+* Remember that document names use ``/`` as the path separator and don't
+  contain the file name extension.
```

Comment:
original sentence has 80 chars, why is it wrapped? it looks as if in fact the entire file was wrapped at shorted length perhaps 78 chars.

Reply:
I'm pretty sure it was 81 but I can check. I was wrapping with vim (`v` to select text then `gq` to wrap).